readme.md

```diff
@@ -99,13 +99,18 @@ console.log("IO is a side effect!");
 
 ## Idempotency
 
-> A function is said to be idempotent if it has no side-effects on multiple
-executions with the the same input parameters.
+> A function is said to be idempotent if there are no additional effects of calling it a second time with the same parameters.
```

Comment:
@CrossEye @hemanth You guys are amazing. And I am totally confused. I don't have math or functional programming background, but I have been prgramming (poorly) for 15+ years. Finally, after discovering ramda my life has changed. 

This is a great effort to educate people like me, and we need to promote functional style. This will help millions of programmers out there, just like me. 

I got totally confused with the idempotent definition, still am. Please continue this effort. 

And please let me know how I can help.